### PR TITLE
Upgrade Telescope to 0.80

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: scripts/codegen.js [hunk 1]
```diff
@@ -111,6 +94,9 @@ telescope({
     `;
     writeFileSync(`${outPath}/index.ts`, index_ts);
 
+    // I think we don't need this one
+    unlinkSync(`${outPath}/extern.ts`);
+
```
Comment (pyramation) [inline scripts/codegen.js:99]: oh yea this is new, I'm realizing, let me just remove that file if it's not specified in the options. I will make it an option and default it to false.

Comment (pyramation) [reply scripts/codegen.js:99]: The goal will be you should be able to upgrade w/o changes like this, my apologies.

Comment (pyramation) [reply scripts/codegen.js:99]:
```
Successfully published:
 - @osmonauts/telescope@0.80.0
 ```
 
[ this commit removed the file](https://github.com/osmosis-labs/telescope/commit/eae54bbc14b6e55a5f7c65b1b1df4dee0aaff6f1)

and now [it's only exported with these options](https://github.com/osmosis-labs/telescope/blob/eae54bbc14b6e55a5f7c65b1b1df4dee0aaff6f1/packages/telescope/src/generators/create-helpers.ts#L31-L36)

Comment (webmaster128) [reply scripts/codegen.js]: Nice, done

**Pull request merged**